src/carousel/carousel.ts

```diff
@@ -84,6 +84,11 @@ export class NgbCarousel implements AfterContentChecked,
    */
   @Input() keyboard: boolean;
 
+  /**
+   * A flag to enable or disable mouse events
```

Comment:
`mouse events` is too generic in here. we only check for mouse enter/leave

I would rephrase it to `Flag to enable pause/resume on mouseover`.